Python/Product/PythonTools/PythonTools/Intellisense/TaskProvider.cs

```diff
@@ -85,19 +86,22 @@ LocationTracker spanTranslator
         public bool IsValid => _squiggle && !string.IsNullOrEmpty(ErrorType);
 
         public void CreateSquiggleSpan(SimpleTagger<ErrorTag> tagger) {
-            if (_rawSpan.Length <= 0 || _spanTranslator == null) {
+            if (_rawSpan.Start <= _rawSpan.End || _spanTranslator == null) {
                 return;
             }
 
-            SnapshotSpan target = _spanTranslator.TranslateForward(
-                new Span(_rawSpan.Start.Index, _rawSpan.Length)
-            );
+            // TODO: Map between versions rather than using the current snapshot
```

Comment:
Did you mean to get back to that before checkin or later?

Reply:
Later